code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/InputBoxNotificationBuilder.kt

```diff
@@ -83,8 +77,7 @@ internal object InputBoxNotificationBuilder {
             expandedLayout.setImageViewBitmap(R.id.expanded_template_image, pushImage)
         } else {
             Log.trace(
-                PushTemplateConstants.LOG_TAG,
```

Comment:
nit: should we import PushTemplateConstants.LOG_TAG like its done in the class where ever it's used so we can just use `LOG_TAG` within logs?

Reply:
Yeap, I can do that